src/components/ActNowPage/useStyles.js

```diff
@@ -170,6 +170,14 @@ const useStyles = makeStyles(
       },
     },
     accountLogout: {},
+    actNowSummary: {
+      paddingBottom: typography.pxToRem(43),
+    },
+    summaryDetails: {
+      [breakpoints.down("xs")]: {
+        display: "none",
+      },
+    },
```

Comment:
Use [`<Hidden mdDown implementation="css">...</Hidden>`](https://material-ui.com/components/hidden/) to hide this using server-side generated CSS.